ili2gpkg/test/java/ch/ehi/ili2gpkg/RtreeSpatialIndexTest.java

```diff
@@ -0,0 +1,293 @@
+package ch.ehi.ili2gpkg;
+
+
+import ch.ehi.ili2db.base.Ili2db;
+import ch.ehi.ili2db.gui.Config;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import java.io.File;
+import java.sql.*;
+import java.util.ArrayList;
+
+public class RtreeSpatialIndexTest {
+    private static final String TEST_OUT="test/data/RtreeSpatialIndex/";
+    private static final String FILENAME_XTF=TEST_OUT+"RtreeSpatialIndex.xtf";
+    private static final String MODEL_XTF="RtreeSpatialIndex";
+    private static final String FILENAME_XTF_GPKG=TEST_OUT+"RtreeSpatialIndex.gpkg";
+    private static final String PRIMARY_KEY_XTF_GPKG="T_Id";
+    private static final String FILENAME_TEST_DATA_GPKG=TEST_OUT+"test_data.gpkg";
```

Comment:
Diese Datei wird nicht erzeugt oder, sondern es wird verifiziert, dass die Funktionen in einer bestehnden Datei registiert werden können (Falls ja: umbennnen z.B. FILENAME_GPKG_EXISTING)?

Reply:
Genau, die Datei existiert bereits und die Funktionen werden auf der Datei getestet.

Ich habe die Konstante entsprechend umbenannt